boxoffice/views/item_collection.py

```diff
@@ -1,35 +1,42 @@
 # -*- coding: utf-8 -*-
 
-from flask import make_response, render_template, jsonify, request, Markup
+from flask import render_template, jsonify, request, Markup
 from pycountry import pycountry
 from coaster.views import load_models
 from coaster.utils import getbool
 from boxoffice import app
-from boxoffice.models import Organization, ItemCollection, Item
-from utils import xhr_only, cors
+from boxoffice.models import Organization, ItemCollection, Item, DiscountPolicy
+from utils import xhr_only, cors, sanitize_coupons
 from boxoffice.data import indian_states
 
 
 def jsonify_item(item):
+    if item.restricted_entry:
+        code_list = request.args.getlist('code') and sanitize_coupons(request.args.getlist('code'))
```

Comment:
This line is confusing. It could be: `code_list = sanitize_coupons(request.args.getlist('code')) if 'code' in request.args else None`

Reply:
Changed